File: lib/_muhasebe/kasiyer_acik/model/kasiyer_acik_model.dart
import 'dart:convert';

import 'package:json_annotation/json_annotation.dart';

import '../../../core/base/base_model.dart';

part 'kasiyer_acik_model.g.dart';

@JsonSerializable()
class KasiyerAcikModel  extends BaseModel{
  final DateTime? tarih;
  final String? kasiyerAdi;
  final String? personelNo;
  final double? acik;
  final double? fazla;

  KasiyerAcikModel(
  this.tarih, 
  this.kasiyerAdi, 
  this.personelNo,
  this.acik, 
  this.fazla);

  Map<String, dynamic> toJson() => _$KasiyerAcikModelToJson(this);
  
  factory KasiyerAcikModel.fromJson(Map<String, dynamic> json) =>
      _$KasiyerAcikModelFromJson(json);
  
  @override
  String myJson() => jsonEncode(toJson());
  
}

File: GiroDashboard.Application/DTOs/Muhasebe/KasiyerAcikModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GiroDashboard.Application.DTOs.Muhasebe
{
    public class KasiyerAcikModel
    {
        public DateTime? Tarih { get; set; }
        public string? KasiyerAdi { get; set; }
        public string? PersonelNo { get; set; }
        public double? Acik { get; set; }
        public double? Fazla { get; set; }
    }
}
